path-manager/app/services/RangeKeyMatches.java

```diff
@@ -0,0 +1,9 @@
+package services;
+
+import com.amazonaws.services.dynamodbv2.document.RangeKeyCondition;
+
+public final class RangeKeyMatches {
+    public static final RangeKeyCondition rangeKeyMatches(String attrName, Object val) {
+        return new RangeKeyCondition(attrName).eq(val);
```

Comment:
does just move the offending code to a different place? Or does IntelliJ somehow understand it here?

Reply:
Yeah, it's 100% valid Java. Just in Scala it finds the `eq` from `AnyRef` in the hierarchy which is a Scala type not present in Java.